Cluster_Display_Use_Case_v2/stpa/telltale2.md

```diff
@@ -0,0 +1,21 @@
+# Telltale Control structure v2
+
+Work-in-progress control structures for the Telltale Use Case
+
+In both diagrams:
+
+* Blue boxes represent data structures exchanged by controllers
+* Dashed lines represent data flow
+* The orange box is an external hardware watchdog implementing the eGas model
+
+## Control structure with framebuffer checking
```

Comment:
Should be "control and data flow diagramm" as we aligned during the WG meeting on Oct-04 8:00 UTC, that is not a control structure as per STPA definition.